### DISCUSSION TITLE
Implement  Adversarial Attacks in ART Library against "Wavelet Integrated CNNs for Noise-Robust Image Classification" models

### DISCUSSION THREAD

**Discussion by Reza99RD** [Trusted-AI/adversarial-robustness-toolbox#2010] [Ideas]
Hello,
I want to evaluate the robustness of [Noise Rubust WaveCNet PyTorch image classification models](https://github.com/LiQiufu/WaveCNet) against adversarial attacks in ART library. I wonder if someone could help me to implement one of these [WaveCNet noise robust models](https://github.com/LiQiufu/WaveCNet/tree/master/models_dwt) in ART PyTorch predefined classifier and then train it on MNIST (or CIFAR10 dataset), and finally attack the classifier with one of ART's different types of attacks to  evaluate robustness of these noise robust models against adversarial attacks?
Thanks a lot in advance.

**Comment by beat-buesser** [comment]
Hi @Reza99RD It's great to hear from you and about your plans. I'll convert the issue into a Discussion to continue.

Have you tried using the WaveCNet models with ART's `PyTorchClassifier`?
